hack/check-docs.sh

```diff
@@ -14,16 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cp docs/content/en/docs/references/cli/index_header docs/content/en/docs/references/cli/_index.md
-go run cmd/skaffold/man/man.go >> docs/content/en/docs/references/cli/_index.md
-
-readonly CLI_CHANGES=`git status -s | grep "docs/" | wc -l`
-
-if [ $CLI_CHANGES -gt 0 ]; then
```

Comment:
This consistency check is important to have in travis.

Reply:
It’s done as a unit test now